src/main/java/org/folio/rest/validator/AccessTypesBodyValidator.java

```diff
@@ -12,19 +13,28 @@
   private static final String INVALID_REQUEST_BODY_TITLE = "Invalid request body";
   private static final String INVALID_REQUEST_BODY_DETAILS = "Json body must contain data.attributes";
 
+  private final int maxNameLength;
+  private final int maxDescriptionLength;
+
+  public AccessTypesBodyValidator(@Value("${kb.ebsco.credentials.access.types.name.length.max:75}") int maxNameLength,
+                                  @Value("${kb.ebsco.credentials.access.types.description.length.max:150}") int maxDescriptionLength) {
+    this.maxNameLength = maxNameLength;
```

Comment:
the max value for name and description length specified in both application.properties and test-application.properties. there is no need to specify default value for them.

Reply:
If smbd delete this property from properties-file than the application will work with these default values. I think it is a plus to fault tolerance.